### ISSUE TITLE
RangeSelector - WCAG 2.5.5

### ISSUE THREAD

**Issue by britt6612** [grommet/grommet#7645]
### Is this something you or your team is interested in contributing a pull request towards?

None

### Expected Behavior

1. RangeSelector slider does not meet 44X44 target size 

- [ ] Update WCAG rule 2.5.5 in DS site to `passed` once PR is complete 

### Actual Behavior

does not pass the above issues

**Comment by jcfilben** [comment]
@britt6612 Could we break this into two separate issues and then make note of what components and WCAG rules need updated on the DS site when these issues get resolved

**Comment by britt6612** [comment]
@jcfilben yes I can break up the issues and this is regarding RangeSelector since thats being used in the DataFilters

**Comment by jcfilben** [comment]
Ah okay I see you're right this is RangeSelector

**Comment by jcfilben** [comment]
Since 2.5.5 is a AAA rule I'm going to mark this as not planned
